lib/commands/profile.js

```diff
@@ -312,7 +312,9 @@ class Profile extends BaseCommand {
     }
 
     log.info('profile', 'Setting two-factor authentication to ' + mode)
-    const challenge = await otplease(this.npm, conf, c => set(info, c))
+    const challenge = await otplease(this.npm, conf, async c => {
+      return await set(info, c)
```

Comment:
I'm not 100% sure this was needed, or was this a style choice?

Reply:
no, this was a... i was logging and opened up the callback choice, reverting